controllers/auth.js

```diff
@@ -0,0 +1,63 @@
+const bcrypt = require('bcryptjs');
+const nodemailer = require('nodemailer');
+const sendGridTransport = require('nodemailer-sendgrid-transport');
+const { validationResult } = require('express-validator');
+
+const User = require('../models/user');
+
+const transporter = nodemailer.createTransport(
```

Comment:
This is not related to any kind of auth controllers. It is used inside a function, so please separate  this logic into another file and then import from that file to use it.